guac/certifier-scorecard.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: page
+permalink: /guac/scorecard-certifier/
+redirect_from: /scorecard-certifier/
+title: "Scorecard certifier"
+parent: "How GUAC components work together"
+description:
+  "Guide to using the Scorecard Certifier in GUAC for security assessment of
+  open source repositories"
+---
+
+# Scorecard Certifier
+
+## Overview
+
+The Scorecard Certifier component of [GUAC](https://guac.sh) (Graph for
+Understanding Artifact Composition) integrates with the
+[OpenSSF Scorecard](https://github.com/ossf/scorecard) project to provide
+comprehensive security risk assessments for open-source repositories. It
+evaluates repositories against industry security best practices and provides
+actionable security insights.
+
+## Key Features
+
+- **Security Risk Assessment**: Evaluates repositories using 19 comprehensive
+  security checks
+
+- **Dual Operation Modes**: Supports both API-based and local scorecard
+  generation
+
+- **Automated Scoring**: Provides numerical scores (0-10) for each security
+  check
+
+- **Comprehensive Analysis**: Covers code review practices, dependency
+  management, vulnerability handling, and more
+
+- **Integration with GUAC**: Seamlessly connects scorecard data to GUAC's
+  software supply chain graph
+
+## Data Collection Process
+
+1. **Repository Identification**:
+   - Source repositories of packages are identified by GUAC
+   - Supports GitHub repositories with commit SHA or tag references
+   - Validates repository metadata and accessibility
+
+2. **Scorecard Evaluation**:
+   - Runs security checks against target repositories
+   - API mode: Fetches pre-computed results from OpenSSF Scorecard API
+   - Local mode: Executes scorecard checks using the scorecard library and
+     computes the result
+
+3. **Data Ingestion**:
+   - Converts scorecard results to structured JSON format
+   - Publishes results through GUAC's event stream for ingestion
+
+## Operation Modes
```

Comment:
This section could use one or two sentences to explain at a high level what the two modes are and why the user should select one or the other.

A very rough suggestion, please feel free to edit it heavily:

> The Scorecard certifier has two modes: API fetcher (default) and local fetcher. API fetcher mode uses the OpenSSF Scorecard API, which is faster but has less coverage. Local fetcher mode is slower, but can evaluate any accessible GitHub repository.